src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -60,6 +60,38 @@ class LocalBackend extends Backend with LazyLogging {
 
   import LocalBackend._
 
+  private class TempFileFilter(prefix: String) extends FilenameFilter {
+    val regex = s"^$prefix.*\\.tmp$$".r
+    def accept(dir: File, name: String): Boolean = regex.findFirstIn(name) match {
+      case Some(s) => true
+      case _ => false
+    }
+  }
+
+  override def stdout(workflowId: WorkflowId, callFqn: FullyQualifiedName): WdlFile = {
+    val fqnParts = callFqn.split("\\.")
+    if (fqnParts.size < 2) {
+      throw new UnsupportedOperationException("FQN not complete")
+    }
+    val hostCallDirectory = hostCallPath(fqnParts.head, workflowId, fqnParts.last).toFile
+    hostCallDirectory.listFiles(new TempFileFilter(prefix="stdout")).toSeq match {
+      case s:Seq[File] if s.size == 1 => WdlFile(s.head.getAbsolutePath)
+      case _ => throw new StdoutStderrException(s"No standard output for call $callFqn")
+    }
+  }
+
+  override def stderr(workflowId: WorkflowId, callFqn: FullyQualifiedName): WdlFile = {
+    val fqnParts = callFqn.split("\\.")
+    if (fqnParts.size < 2) {
+      throw new UnsupportedOperationException("FQN not complete")
+    }
+    val hostCallDirectory = hostCallPath(fqnParts.head, workflowId, fqnParts.last).toFile
```

Comment:
I think this was already referred to but abstract this file oriented stuff as well as it's nearly identical to the one above